### ISSUE TITLE
Prebuild show-case app and commit to repo

### ISSUE THREAD

**Issue by MeirionHughes** [aurelia/app-ux-showcase#19]
There is currently no online show-case for people to see the running app. 

Its easier to sell the UX to people if they can try it without effort. There should be a link off the github readme a working UX show-case application. Webpack is very effective for bundling and I have a working example ("try example") to show-case custom plugins: https://github.com/MeirionHughes/aurelia-plugin-skeleton

**Comment by ZHollingshead** [comment]
Agreed. https://github.com/aurelia/app-ux-showcase/issues/4 mentions this as well and I believe this is still going to happen.

@EisenbergEffect probably has more information on that though.

**Comment by EisenbergEffect** [comment]
Hoping to drop this into the new developer Hub once we get that a little farther along.

**Comment by khuongduybui** [comment]
6 weeks later, are we any closer to getting this done?

**Comment by EisenbergEffect** [comment]
Hub slowed down due to major conferences and other items on the agenda. So, no, not any closer.

**Comment by enrico-padovani** [comment]
It's a shame to not be able to quickly evaluate components for devs and decision makers! Today I was asked about a ui framework for vue and in 30 secs I found that awesome components and docs: http://element.eleme.io/#/en-US/component/quickstart. I think Aurelia needs to be more accessible for new developers in order to increase its community.

**Comment by avrahamcool** [comment]
any news on this?

**Comment by Alexander-Taran** [comment]
https://aux-demo.firebaseapp.com/introduction

same as https://github.com/aurelia/app-ux-showcase/issues/4

**Comment by brianjlacy** [comment]
The demo link provided does not work for me. Just a blank screen.

**Comment by rhux** [comment]
@brianjlacy ... the blank screen appears to be an issue with webpack 4.2 see here (https://github.com/aurelia/cli/issues/852)

The fix is to run: `au run --env dev --watch`

**Comment by veikkoeeva** [comment]
@brianjlacy A working link seems to be https://aux-demo.firebaseapp.com/getting-started.
